test/tests/functional/pbs_mom_mock_run.py

```diff
@@ -58,13 +58,15 @@ def test_rsc_used(self):
 
         # Submit a job requesting ncpus, mem and walltime
         attr = {ATTR_l + ".select": "1:ncpus=1:mem=5mb",
-                ATTR_l + ".walltime": "00:00:05"}
+                ATTR_l + ".walltime": "00:00:20"}
         j = Job(attrs=attr)
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
 
-        # This job should end in 5 seconds, let's sleep
-        time.sleep(7)
+        # This job should end in 20 seconds, let's sleep
+        msg = "This job should end in 20 seconds, let's sleep"
+        self.logger.info(msg)
+        time.sleep(22)
```

Comment:
I'd shorten the message so it'd fit on one line with the logger.  You can won't need to say how long you are sleeping if you replace the sleep with self.server.expect(JOB, 'queue', op=UNSET, id=jid, expect=22)